es-ds-docs/components/ds-molecules-list.vue

```diff
@@ -72,5 +72,8 @@
         <li>
             <ds-link to="/molecules/view-more"> View more </ds-link>
         </li>
+        <li>
+            <ds-link to="/molecules/verification-code"> Verification Code </ds-link>
+        </li>
```

Comment:
Nit: Alphabetically, this should come before "View more" in the list.

Also: Can we use sentence case to be consistent with the other list items, so it's "Verification code" rather than "Verification Code"?